plugins/must-gather/commands/comprehensive-analysis.md

```
@@ -0,0 +1,795 @@
+├── analyze_pvs.py
+├── analyze_ingress.py
+├── analyze_machineconfigpools.py
+├── analyze_servicelogs.py
+├── analyze_pod_logs.py
+├── analyze_node_logs.py
+└── analyze_ovn_dbs.py
+```
+
+## Error Handling
+
+**CRITICAL: Script-Only Analysis**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)